logstash-core/lib/logstash/api/commands/stats.rb

```diff
@@ -92,9 +108,13 @@ def hot_threads(options={})
         end
 
         private
-        def plugins_stats_report(pipeline_id)
+        def plugins_stats_report(pipeline_id, opts)
```

Comment:
This method is private, so we _should_ be able to find and modify all call-sites, but a non-breaking equivalent signature would be:

```suggestion
        def plugins_stats_report(pipeline_id, opts={})
```